packages/node/src/indexer/unfinalizedBlocks.service.ts

```diff
@@ -36,11 +36,6 @@ export class UnfinalizedBlocksService extends BaseUnfinalizedBlocksService<
     super(nodeConfig, storeCache);
   }
 
-  @mainThreadOnly()
```

Comment:
We should also remove `substrateHeaderToHeader` function

Reply:
Beside this place been used, I think a few other places is using is it
1. https://github.com/subquery/subql/blob/6a4d55fe0e7e727b790bca650799adc6a476d817/packages/node/src/indexer/fetch.service.ts#L68
2. https://github.com/subquery/subql/blob/6a4d55fe0e7e727b790bca650799adc6a476d817/packages/node/src/indexer/unfinalizedBlocks.service.ts#L51